packages/adapter/src/response-processing/index.ts

```diff
@@ -0,0 +1,43 @@
+export * from './casting';
+export * from './encoding';
+export * from './extraction';
+
+// export function isNumberType(type: ResponseType) {
```

Comment:
What's up with the commented out code?

Reply:
Woops I missed that. It was supposed to be removed